helm_helpers.go

```diff
@@ -610,43 +323,36 @@ func cleanUntrackedReleases() {
 	}
 
 	if len(toDelete) == 0 {
-		log.Println("INFO: no untracked releases found.")
+		logs.Info("No untracked releases found.")
```

Comment:
Upper-case first letter for logs seems inconsistent with other logs.Info

Reply:
:+1: